torch/nn/modules/module.py

```diff
@@ -157,11 +157,16 @@ def register_module_backward_hook(
     _global_backward_hooks[handle.id] = hook
     return handle
 
-
-# Trick mypy into not applying contravariance rules to inputs by defining
-# forward as a value, rather than a function.  See also
-# https://github.com/python/mypy/issues/8795
 def _forward_unimplemented(self, *input: Any) -> None:
+    r"""Forward defines the computation performed at every call.
```

Comment:
Instead of making this a docstring, I would have expected the string next to forward to be set as the docstring on the forward object and for this to be unchanged. Don't you want the other string as the docstring?

Reply:
thanks for the advice. I wasn't sure how best to do this as I was reluctant to move the docstring for `forward` away from the definition of `forward`. 

I've made that change.